client/src/templates/Challenges/redux/execute-challenge-saga.js

```diff
@@ -99,17 +99,10 @@ export function* executeChallengeSaga({ payload }) {
     yield put(updateTests(testResults));
 
     const challengeComplete = testResults.every(test => test.pass && !test.err);
-    const playSound = store.get('fcc-sound');
-    let player;
-    if (playSound) {
-      void import('tone').then(tone => {
-        player = new tone.Player(
-          challengeComplete && payload?.showCompletionModal
-            ? 'https://campfire-mode.freecodecamp.org/chal-comp.mp3'
-            : 'https://campfire-mode.freecodecamp.org/try-again.mp3'
-        ).toDestination();
-        player.autostart = true;
-      });
+    if (challengeComplete && payload?.showCompletionModal) {
```

Comment:
```suggestion
    if (challengeComplete) {
```

Okay so I ran in to this with the new multi-file editor.

If you click the button to run the tests, they can all pass but the completion modal doesn't show so it plays the error sound instead.